.eslintignore

```diff
@@ -11,6 +11,7 @@
 /tests/screenshot/*
 /tests/test_runner.js
 /tests/workspace_svg/*
+/tests/typescript/*
```

Comment:
I'm not sure how straightforward it will be, though I'll look further into it this upcoming Friday. And get back to you when I've looked into it a bit more.

The configuration in `.eslintrc.json` points to the root `./tsconfig.json` via  `parserOptions` which I believe was the main issue, though there may be around that. These tests need their own tsconfig so they aren't built and packaged with the rest of the code.

Reply:
Updated! The tsconfig is defined in an array, which I hadn't noticed at my quick glance earlier, so I just had to add the second override.

Of note, I updated `.eslintrc.json` to `.eslintrc.js` since we likely want the same settings for the overrides, and it's easier to keep them in sync via JS than in JSON.